index.html

```diff
@@ -11,232 +11,311 @@
+                </li>
+            </ul>
+        </nav>
     </header>
-    <section>
-        <div class="self-presentation">
-            <img src="images/image.png" alt="Ups, the image is broken" class="girl-full-stack">
-            <p class="right-text">
-                I began my IT career two years ago as a Manual QA engineer student.
-                Since then, I've worked as a manual QA on a mobile project,
-                transitioned to a general QA role, and have now become an AQA.
-                Nowadays, I work with end-to-end testing using the Playwright framework and TypeScript,
-                API testing, and manual testing of web and mobile platforms.
-                <span class="color-text">I love what I do.</span>
-            </p></br>
-        </div>
-    </section>
-
-    <section>
-        <div class="arrow-container">
-            <img src="images/down-arrow.png" alt="Ups, the image is broken" class="down-arrow">
-        </div>
-    </section>
-
-    <section>
-        <div>
-            <h2 class="centered-text italic">Set of my skills</h2>
-        </div>
-        <div>
-            <table class="table">
-                <tr class="main-row color-text">
-                    <th>Manual Mobile and Web App QA</th>
-                    <th>API QA</th>
-                    <th>AQA</th>
-                </tr>
-                <tr class="table-section-header">
-                    <th>Testing Fundamentals</th>
-                    <th>API Testing Fundamentals</th>
-                    <th>Automation Frameworks</th>
-                </tr>
-                <tr>
-                    <td>
-                        <ul>
-                            <li>Knowledge of SDLC and STLC</li>
-                            <li>Proficiency in writing and working with test documentation (test cases, checklists)</li>
-                            <li>Understanding of different types of testing: functional, non-functional</li>
+    <main>
+        <section id="about_me">
+            <div class="about-me">
+                <img src="images/image.png" alt="Ups, the image is broken" class="about-me__image">
+                <div class="about-me__description">
+                    <h1 class="about-me__header">Hi, I'm Nataliia. Nice to see you in my place! Enjoy!</h1>
+                    <p class="about-me__paragraph">
+                        I began my IT career two years ago as a Manual QA engineer student.
+                        Since then, I've worked as a manual QA on a mobile project,
+                        transitioned to a general QA role, and have now become an AQA.
+                        Nowadays, I work with end-to-end testing using the Playwright framework and TypeScript,
+                        API testing, and manual testing of web and mobile platforms.
+                        <span class="color-text">I love what I do.</span>
+                    </p></br>
```

Comment:
Start moving away from br. Its not used in modern development. You can achieve the same results using multiple tags or with flex.